lib/torrent.js

```diff
@@ -509,6 +505,22 @@ Torrent.prototype._onMetadata = function (metadata) {
     return new File(self, file)
   })
 
+  // Only select specified files (BEP53)
+  if (self.so) {
+    var specificFiles = parseRange.parse(self.so)
+    // https://github.com/webtorrent/webtorrent/issues/164
+    self.deselect(0, self.pieces.length - 1, false)
+
+    self.files.forEach(function (v, i) {
+      specificFiles.indexOf(i) === -1 ? self.files[i].deselect() : self.files[i].select(true)
```

Comment:
Unless I'm missing something, it shouldn't be necessary to call `deselect()` since all files start out deselected. Can we change this to just call `select()`?

Also, can you using `array.includes()` instead of `array.indexOf()` since it's clearer?

Reply:
You're correct, the `self.files[i].deselect()` serves no purpose as pieces start out deselected, this will be removed in the next commit.

Replacing `specificFiles.indexOf(i)` with `selectOnlyFiles.includes(i)` for clarity will also be in the next commit.

Thank you for the review :)